.github/workflows/test.yml

```diff
@@ -26,16 +26,16 @@ jobs:
 
       - name: Setup
         run: |
-          wget -q http://downloads.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.0.2.tgz
-          tar xf mongodb-linux-x86_64-ubuntu1804-4.0.2.tgz
+          wget -q https://downloads.mongodb.org/linux/mongodb-linux-x86_64-ubuntu2004-4.4.6.tgz
```

Comment:
Is it necessary to upgrade to 4.4 in this change if mongodb-topology-manager doesn't support it? I'd rather remove mongodb-topology-manager in a separate release.

Reply:
According to https://www.mongodb.com/download-center/community/releases/archive, 4.4.0 is the first version with a Ubuntu 20.04 archive/package. 
I'll revert `test.yml` for now.